plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/glue/IcebergGlueCatalogConfig.java

```diff
@@ -32,17 +30,4 @@ public IcebergGlueCatalogConfig setCacheTableMetadata(boolean cacheTableMetadata
         this.cacheTableMetadata = cacheTableMetadata;
         return this;
     }
-
-    public boolean isSkipArchive()
-    {
-        return skipArchive;
-    }
-
-    @Config("iceberg.glue.skip-archive")
```

Comment:
Remove also from the `metastores.md` if the decision is to remove.
Are we talking about a "breaking change" if we remove the property?

Reply:
Done! It's not a breaking change, as the old property will still be recognized.